src/util/Filter.js

```diff
@@ -42,7 +42,7 @@ let categoryMap = new Map([
 ]);
 
 const operators = ['>=', '<=', '<', '>', ':', '!=', '='];
-const operatorsRegex = new RegExp('(?:' + operators.join('|') + ')');
+export const operatorsRegex = new RegExp('(?:' + operators.join('|') + ')');
```

Comment:
we should only be using export once at the end of the file, if these functions should be accessible outside then add them to the object export at the end of the file

Reply:
I've used this pattern elsewhere - no problem in ES6 modules with mixing named + default exports

Reply:
I should also note that without specific exports like this one you _can't_ use the `import { operatorsRegex } from 'util/Filter'` pattern. The ES6 imports we use in the client code work differently from the commonjs ones in the server code.